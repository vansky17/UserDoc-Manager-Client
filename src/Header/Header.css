html, body{
  margin: 0;
}
#home {
    padding: 10% 2%;
     background:
    linear-gradient(
      rgba(0, 0, 0, 0.5), 
      rgba(0, 0, 0, 0.8)
    );
   /* url(https://cdn.pixabay.com/photo/2019/02/10/17/10/notebook-3987701_960_720.jpg) repeat;  */  
   /*  url(https://cdn.pixabay.com/photo/2018/03/07/07/55/laptop-3205413_960_720.jpg) repeat; */ 
    /* url(https://cdn.pixabay.com/photo/2017/09/05/10/07/office-2717006_960_720.jpg) repeat;  */
    /* url(https://cdn.pixabay.com/photo/2018/03/16/04/13/composition-3230164_960_720.jpg) repeat;  */
    color: rgba(9, 182, 236, 1);
    text-shadow: 2px 2px #000;
    background-size: cover;
    background-position: center; 
  }

  #home h1{
    font-size: 2em;
    margin: 0.67em 0;
  }
  #home h3, h4{
    font-weight: 300;
    line-height: 1.5em;
  }
  .video-container {
    position: relative;
    /* height: 100vh;
    width: 100vw; */
    overflow: hidden;
  }
  .video-container video {
    z-index: -1;
    position: absolute;
    width: auto;
    height: auto;
    min-width: 100%;
    min-height: 100%;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
  }