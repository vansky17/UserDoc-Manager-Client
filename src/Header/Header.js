import React from 'react'
import './Header.css'

class Header extends React.Component {
  render () {
    return (
      
      <header id="home" role="banner" className="video-container">
        
          <h1 >UserDocs Manager</h1>
          <h3 >Overview Of Technical Documents</h3>
          <h4 >Your Department</h4>
        
      </header>
      
   
    )
  }
}

export default Header