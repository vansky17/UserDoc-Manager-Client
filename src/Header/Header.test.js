import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './Header';
import ReactDOM from 'react-dom';


describe(`Header component`, () => {
  const props = {
    id: 'a',
    className: 'test-class-name',
  };

  it('renders a .Header by default', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('renders the Header given props', () => {
    const wrapper = shallow(<Header {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
describe("Header renders properly", () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
       <Header />,
      div
    );
    ReactDOM.unmountComponentAtNode(div)
  });
});