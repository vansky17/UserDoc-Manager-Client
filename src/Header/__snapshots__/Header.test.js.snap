// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders a .Header by default 1`] = `
<Fragment>
  <header
    id="home"
    role="banner"
  >
    <h1>
      UserDocs Manager
    </h1>
    <h3>
      Overview Of Technical Documents
    </h3>
    <h4>
      Your Department
    </h4>
  </header>
</Fragment>
`;

exports[`Header component renders the Header given props 1`] = `
<Fragment>
  <header
    id="home"
    role="banner"
  >
    <h1>
      UserDocs Manager
    </h1>
    <h3>
      Overview Of Technical Documents
    </h3>
    <h4>
      Your Department
    </h4>
  </header>
</Fragment>
`;
